src/ReadVTK.jl

```diff
@@ -126,10 +130,18 @@ function VTKFile(filename)
   # Ensure supported compression type
   @assert in(compressor, ("", "vtkZLibDataCompressor"))
 
-  # Extract piece
-  piece = root[file_type][1]["Piece"][1]
-  n_points = parse(Int, attribute(piece, "NumberOfPoints", required=true))
-  n_cells = parse(Int, attribute(piece, "NumberOfCells", required=true))
+  if file_type == "UnstructuredGrid"
+    # Extract piece
+    piece = root[file_type][1]["Piece"][1]
+    n_points = parse(Int, attribute(piece, "NumberOfPoints", required=true))
+    n_cells = parse(Int, attribute(piece, "NumberOfCells", required=true))
+  elseif file_type == "ImageData"
+    dataset_element = root[file_type][1]
+    wholeExtent = parse.(Int, split(attribute(dataset_element, "WholeExtent", required=true), ' '))
+    n_points_per_grid_dir = [wholeExtent[2*i]+1 for i in (1:3)]
```

Comment:
```suggestion
    whole_extent = parse.(Int, split(attribute(dataset_element, "WholeExtent", required=true), ' '))
    n_points_per_grid_dir = [whole_extent[2*i]+1 for i in (1:3)]
```
Variables/functions should be snake case